src/include/rasta/rastafactory.h

```diff
@@ -17,7 +17,7 @@ extern "C" { // only need to export C interface if
  */
 struct RastaConnectionData {
     unsigned short send_max;
-    char version[4];
+    char version[5];
```

Comment:
Okay, the standard says that the protocol version is four bytes, but we interpret it as a string and therefore have to add a fifth byte to accomodate the terminating null (i.e. to be able to use strcpy).

However, is this correct? 

<img width="487" alt="Screenshot 2023-06-12 at 09 21 29" src="https://github.com/eulynx-live/rasta-protocol/assets/2394156/57e46443-a309-44d2-bcff-14a3b874a338">

yup. 👍